docs/index.html

```diff
@@ -9,8 +9,8 @@
   <style>
     main {
       width: 80%;
-      margin: var(--auro-size-xl) auto var(--auro-size-xxxl);
-      padding-bottom: var(--auro-size-xxxl);
+      margin: var(--ds-size-400) auto var(--ds-size-800);
```

Comment:
```suggestion
      margin: var(--ds-size-400, $ds-size-400) auto var(--ds-size-800, $ds-size-800);
```

Please resolve all missing fallback token references.

Reply:
oh wait, this one is html file, SCSS variables couldn't work here, isn't it?

Reply:
Oops, that's my bad. Knee-jerk response. Nevermind.